okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -43,18 +47,27 @@
 import static org.junit.Assert.fail;
 
 public final class EventListenerTest {
-  @Rule public final MockWebServer server = new MockWebServer();
```

Comment:
There is currently a dependency when using the `SocksProxy` (I added a test that uses it) to shutdown the MWS first. Alternatively, SocksProxy could be adjusted to behave like MWS by closing all open connections.